diracx-routers/src/diracx/routers/auth/token.py

```diff
@@ -158,7 +157,7 @@ async def get_oidc_token(
             detail=str(e),
             headers={"WWW-Authenticate": "Bearer"},
         ) from e
-    except JoseError as e:
+    except (DecodeError, ExpiredTokenError) as e:
```

Comment:
I feel like these should be treated differently, e.g. 

```suggestion
    except (ExpiredTokenError) as e:
        raise HTTPException(
            status_code=status.HTTP_401_UNAUTHORIZED,
            detail=f"Expired JWT: {e}",
        ) from e
    except (DecodeError) as e:
        raise HTTPException(
            status_code=status.HTTP_400_BAD_REQUEST,
            detail=f"Invalid JWT: {e}",
        ) from e
```